README.md

```diff
@@ -2328,6 +2328,21 @@ class API < Grape::API
 end
 ```
 
+For using Rack::Sendfile middleware, specify `to_path` method in your
```

Comment:
Ok I get it. I think it should look like the text above, which is _If you want a file-like object to be streamed using Rack::Chunked, use stream._

Maybe 

```
If you want to take advantage of Rack::Sendfile, which intercepts responses whose body is being served from a file and replaces it with a server specific X-Sendfile header, ...
```

Update it as you see fit and I'll merge.